packages/menu-bar/test/dom/__snapshots__/menu-bar.test.snap.js

```diff
@@ -59,7 +59,6 @@ snapshots["menu-bar basic"] =
 snapshots["menu-bar overlay"] = 
 `<vaadin-menu-bar-overlay
   opened=""
-  right-aligned=""
```

Comment:
Something was wrong about this snapshot having both `right-aligned` and `start-aligned`, seems like the `right-aligned` was set by `__alignOverlayPosition()` from `ContextMenuMixin` since we used to delay setting `positionTarget` (now it's set synchronously so the above logic doesn't run).